packages/backend/src/config/env.ts

```diff
@@ -7,12 +8,16 @@ const envSchema = z.object({
   JWT_ACCESS_TOKEN_SECRET: z.string(),
   JWT_ACCESS_TOKEN_EXPIRATION_TIME: z.coerce.string(),
   JWT_REFRESH_TOKEN_SECRET: z.string(),
-  JWT_REFRESH_TOKEN_EXPIRATION_TIME: z.coerce.number()
+  JWT_REFRESH_TOKEN_EXPIRATION_TIME: z.coerce.number(),
+  RAPYD_API: z.string(),
+  RAPYD_ACCESS_KEY: z.string(),
+  RAPYD_SECRET_KEY: z.string()
 })
 
 type ENV = z.infer<typeof envSchema>
 
 const env: ENV = (() => {
+  dotenv.config()
   const result = envSchema.safeParse(process.env)
```

Comment:
I'd comment the dotenv import and use when merging into main. We are expecting everything to run on the docker environment variables being set.